src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <div class="container">
+        <div class="add-list">
+            <h1>Today's To Do</h1>
+                <div class="input-field">
+                    <input type="text" placeholder="Add your list...">
+                    <button class="submit-btn">Add</button>
+                </div>
+        </div>
+
+        <div class="generated-list">
+
+        </div>
+        
+    </div>
```

Comment:
- [ ] Kindly try to make your design a clone from the guidelines video as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Your%20list%20should%20be%20a%20clone%20of%20the%20part%20of%20the%20minimalist%20project%20captured%20in%20the%20video%20below.).